backend/static-db/src/users/dtos/post/create.dto.ts

```diff
@@ -0,0 +1,15 @@
+import { IsNotEmpty, IsString, IsNumber, IsDateString } from 'class-validator';
+
+export class CreatePostDTO {
+  @IsNotEmpty()
+  @IsNumber()
+  readonly moneyGoal: number;
```

Comment:
moneyGoal should be nullable, since a business user may not have a particular money amount to raise.

Reply:
Done